server/resources.py

```diff
@@ -0,0 +1,25 @@
+from flask_restful import Resource, Api, abort
+from .models import Comment, Lecture
+
+api = Api()
+
+class CommentListResource(Resource):
+    def get(self, lecture_id):
+        db_lectures = Lecture.query.filter(Lecture.id == lecture_id).all()
+
+        if not db_lectures:
+            abort(404, message="Lecture {} does not exist".format(lecture_id))
+
+        db_comments = Comment.query.filter(Comment.lecture_id == lecture_id)
+
+        comments = [{
+            'id': c.id,
+            'content': c.content}
+            for c in db_comments]
```

Comment:
Yes, I was really unsure about this. But what if the object I'm creating gets bigger? It would then become a very long line. Any better way of doing it?